cmd/minikube/cmd/config/config.go

```diff
@@ -163,6 +163,10 @@ var settings = []Setting{
 		name: "native-ssh",
 		set:  SetBool,
 	},
+	{
+		name: "kube-binaries-mirror",
```

Comment:
I suggest putting this in cluster config struct instead of global config
So we can have multiple clusters with different mirrors and we could test in integration tests

Reply:
Fixed.